AIServer/Define.h

```diff
@@ -3,13 +3,16 @@
 //
 //	Defines About Communication
 //
-#define MAX_SOCKET					100
-#define MAX_PATH_LINE		100
+#define MAX_SOCKET				100
+#define MAX_PATH_LINE			100
+
+#define MAX_NPC_SIZE			30
+#define MAX_SEND_USERID			10
+#define MAX_SEND_NPCID			100
 
-#define MAX_NPC_SIZE		30
 #define MAX_WEAPON_NAME_SIZE	40
-#define VIEW_DIST			48		// ���ðŸ�
-#define MAX_UPGRADE_WEAPON	12
+#define VIEW_DIST				48		// ���ðŸ�
+#define MAX_UPGRADE_WEAPON		12
```

Comment:
* The project uses C++17, so I personally use compiled time types, rather than letting the compiler choose what type this will be. e.g.:
  ```cpp
  constexpr uint8_t VIEW_DIST = 48;
  ```

* Please also use `SPACE`s rather than `TAB`s. Some editors could render them differently, which is why using `TAB`s could be inconsistent at times. As you can see it looks strange on GitHub's diffs.